### PR TITLE
youtube-dl: add Polish translation

### PR DESCRIPTION
**Pull request by chanibal**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pl/common/youtube-dl.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`youtube-dl --format {{mp4}} -o "{{%(title)s by %(uploader)s on %(upload_date)s in %(playlist)s.%(ext)s}}" '{{url}}'`
+
+- Pobierz plik razem z napisami:
+
+`youtube-dl --sub-lang {{en}} --write-sub '{{https://www.youtube.com/watch?v=Mwa0_nE9H7A}}'`
+
+- Pobierz ścieżkę dźwiękową ze wszystkich filmów z playlisty:
+
+`youtube-dl -i --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" '{{url to playlist}}'`
```
Comment (navarroaxel) [inline pages.pl/common/youtube-dl.md:36]: We use `snake_case` for variable names:
```suggestion
`youtube-dl -i --extract-audio --audio-format mp3 -o "%(title)s.%(ext)s" '{{url_to_playlist}}'`
```
You can include the fix to the English page in this PR if you want.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @chanibal, I made 1 comment.

File: pages.pl/common/youtube-dl.md (hunk 1 above)
Comment (chanibal) [reply pages.pl/common/youtube-dl.md]: Commited the change in both Polish and English @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @chanibal!

**Pull request merged**